traffic_ops/app/db/migrations/20200414000000_interfaces.sql

```diff
@@ -0,0 +1,151 @@
+/*
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+-- +goose Up
+CREATE TABLE interface (
+	max_bandwidth bigint DEFAULT NULL CHECK (max_bandwidth IS NULL OR max_bandwidth >= 0),
```

Comment:
Blueprint specifies that the default for max_bandwidth should be 0

Reply:
Yeah, the intention was for the default to be "no limit", but since writing the blueprint the sentiment has been that instead `0` should literally just mean `0` and `null` should mean "no limit".